Week1/homework/app.js

```diff
@@ -1,11 +1,153 @@
+    },
+    girl_train: {
+      title: 'The Girl on the Train',
+      language: 'english',
+      author: 'Paula Hawkins',
+    },
+    habits_highly_effective_people: {
+      title: 'The 7 Habits of Highly Effective People: Powerful Lessons in Personal Change',
+      language: 'english',
+      author: 'Stephen R. Covey',
+    },
+    philosophy_fear: {
+      title: 'A Philosophy of Fear',
+      language: 'english',
+      author: 'Lars Fredrik Händler Svendsen',
+    },
+    sapiens: {
+      title: 'Sapiens: A Brief History of Humankind',
+      language: 'english',
+      author: 'Yuval Noah Harari',
+    },
+    lust_life: {
+      title: 'Lust for Life',
+      language: 'english',
+      author: 'Irving Stone',
+      version: 2,
+    },
+  };
+  const bookCover = {
+    parallel_worlds:
+      'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1348224200i/873687.jpg',
+    masnavi:
+      'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1171037882i/83264._SY475_.jpg',
+    black_holes_baby_universes:
+      'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1386924312i/53200.jpg',
+    blindness:
+      'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1327866409i/2526.jpg',
+    one_Hundred_years_solitude:
+      'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1327881361i/320.jpg',
+    girl_train:
+      'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1490903702i/22557272.jpg',
+    habits_highly_effective_people:
+      'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1421842784i/36072.jpg',
+    philosophy_fear:
+      'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1328701163i/5800070.jpg',
+    sapiens:
+      'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1420585954i/23692271.jpg',
+    lust_life:
+      'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1348532272i/79834.jpg',
+  };
+
+  function makeList() {
+    const body = window.document.getElementById('body');
+    const ul = window.document.createElement('ul');
+    // I think there is no need to add id to lists, because we can find it just by item name at first I used this codes but now I comment it:
+    // li.setAttribute('id', item);
+    // console.log(bookCover[item]);
+
+    for (const item in bookInfo) {
+      if (item) {
+        const li = window.document.createElement('li');
+        const image = window.document.createElement('img');
+        image.setAttribute('src', bookCover[item]);
+        li.appendChild(image);
+
+        ul.appendChild(li);
+        const h3Title = window.document.createElement('h3');
+        h3Title.innerHTML = bookInfo[item].title;
+        li.appendChild(h3Title);
+
+        const bookProperties = Object.keys(bookInfo[item]);
+        for (const itemChild in bookProperties) {
```

Comment:
Aahaan. `bookProperties` variable is an array rather than an `object`. We use `for-in` loops for iteration over an objects. You could have made use of `forEach` for a regular `for-of` loop here instead. 🙂